spyder/plugins/completion/plugin.py

```diff
@@ -381,12 +383,33 @@ def on_mainwindow_visible(self):
             provider_info['instance'].on_mainwindow_visible()
 
     # ---------------------------- Status bar widgets -------------------------
-    def register_statusbar_widgets(self):
+    def register_statusbar_widgets(self, plugin_loaded=True):
         """Register status bar widgets for all providers with the container."""
```

Comment:
Please expand this docstring to explain what `plugin_loaded` does (using the Numpydoc format, of course).